daq/runner.py

```diff
@@ -523,10 +525,11 @@ def _handle_device_learn(self, target_mac, vid):
         else:
             self._target_set_trigger(device)
 
-    def _remote_trigger(self, device, device_vlan, assigned_vlan):
+    def _remote_trigger(self, device, device_vlan, assigned_vlan, device_port):
         assert device_vlan, 'expected device_vlan'
         device.port.flapping_start = 0
         device.port.active = True
+        device.nonof_port_num = device_port
```

Comment:
The _device_ entry shouldn't care about OF vs. non-OF... this should just be setting the device port in all cases.

Reply:
device.port.port_no is relied upon to indicate a port trigger as mentioned in the comment above. this was a work_around around it.